modules/mono/glue/cs_files/Mathf.cs

```diff
@@ -10,14 +10,15 @@ namespace Godot
 {
     public static class Mathf
     {
-        // Define constants with Decimal precision and cast down to double or float. 
-        public const real_t PI = (real_t) 3.1415926535897932384626433833M; // 3.1415927f and 3.14159265358979
+        // Define constants with Decimal precision and cast down to double or float.
 
-        #if REAL_T_IS_DOUBLE
-        public const real_t Epsilon = 1e-14; // Epsilon size should depend on the precision used.
-        #else
-        public const real_t Epsilon = 1e-06f;
-        #endif
+        public const real_t TAU = (real_t) 6.2831853071795864769252867666M; // 6.2831855f and 6.28318530717959
+        public const real_t  PI = (real_t) 3.1415926535897932384626433833M; // 3.1415927f and 3.14159265358979
+        public const real_t   E = (real_t) 2.7182818284590452353602874714M; // 2.7182817f and 2.718281828459045
+        public const real_t RT2 = (real_t) 1.4142135623730950488016887242M; // 1.4142136f and 1.414213562373095
+        public const real_t INF = real_t.PositiveInfinity;
+        public const real_t NAN = real_t.NaN;
+        public const real_t EPSILON = real_t.Epsilon;
```

Comment:
Looking into this closer... `real_t.Epsilon` is vastly different from the previously defined Epsilon. `float.Epsilon` is `1.401298e-45f` which is vastly smaller than the previously defined one: `1e-06f`

`Mathf.Epsilon` is used for error-checking in functions like `Vector3.IsNormalized`, but `float.Epsilon` is the smallest possible value. Changing `Mathf.Epsilon` to `real_t.Epsilon` completely breaks this function and causes `IsNormalized` to always return false.